_posts/2024-08-09-contributing-to-eclipse-che-docs.adoc

```diff
@@ -0,0 +1,102 @@
+---
+title: Contributing to Eclipse Che documentation
+layout: post
+author: Jana Vrbkova
+description: >-
+  Contribute to Eclipse Che documentation using an Eclipse Che workspace.
+categories: []
+keywords: ['che-docs', 'documentation', 'write', 'contribute']
+slug: /@deerskindoll/contributing-to-che-docs
+---
+
+== The quick and easy way to write Eclipse Che documentation
+
+Writing about Eclipse Che is even easier than you think.
+The link:https://github.com/eclipse-che/che-docs[Eclipse Che documentation repository on GitHub] was created with community contribution in mind,
+featuring tools you need to create and test your content.
+And from the initial draft to submitting a PR,
+you can do it all in the environment you're already familiar with: an Eclipse Che cloud development environment (CDE).
+
+image::/assets/img/contributing-to-eclipse-che-docs/che-docs.png[An Eclipse Che CDE created from the che-docs repository.]
+
+
+== Getting started
+
+Click the Developer Workspace button in the che-docs repository README file.
+This opens a clone of the repository in an Eclipse Che workspace hosted
+by Red Hat in the link:https://developers.redhat.com/developer-sandbox[Developer Sandbox].
+Alternatively, you can click link:https://workspaces.openshift.com#https://github.com/eclipse/che-docs[*right here*].
+
+== How does it work?
+
+What about the che-docs repository makes using an Eclipse Che CDE to contribute content so convenient? The following two elements:
+
+* link:https://eclipse.dev/che/docs/stable/end-user-guide/devfile-introduction/[devfile]
+* list of link:https://eclipse.dev/che/docs/stable/end-user-guide/microsoft-visual-studio-code-open-source-ide/#automating-installation-of-microsoft-visual-studio-code-extensions-at-workspace-startup[automatically installed Microsoft Visual Studio Code extensions]
+
+Let's look at the devfile first.
+The che-docs repository contains a simple devfile
+that defines the basic features of the che-docs workspace.
+It also includes a command for building a preview of your newly created content:
+
+[source, code]
+----
+# sh tools/preview.sh
+----
+
+This is helpful not only for checking the layout of the document but also for making sure that embedded images,
+links and code blocks are displayed correctly.
+
+By default, the che-docs workspace launches with Visual Studio Code - Open Source ("Code - OSS") as the IDE.
+This is relevant for the second file mentioned above.
+The che-docs repository features an `extensions.json` file listing Visual Studio Code extensions
+you need for writing Eclipse Che content.
+When you launch the che-docs workspace for the first time,
+these extensions are installed automatically.
```

Comment:
@deerskindoll could you please add a screenshot here ?
<img width="347" alt="Screenshot 2024-08-07 at 19 24 54" src="https://github.com/user-attachments/assets/bc34d629-5be4-4844-bab3-7427ae2ff4f3">

Reply:
done